server/routes/products.js

```diff
@@ -14,8 +14,8 @@ let ax = axios.create({
 // root ENDpoint route
 router.route('/')
   .get((req, res) => {
-    req.page = req.query.page || 1;
-    req.count = req.query.count || 5;
+    req.page = req.query.page || '1';
```

Comment:
template literals will coerce everything to a string
not sure what I was trying to accomplish here tbh